websites/serializers.py

```diff
@@ -261,3 +268,72 @@ class Meta:
             "file",
             "is_page_content",
         ]
+
+
+class WebsiteCollectionSerializer(serializers.ModelSerializer):
+    """Serializer for WebsiteCollections"""
+
+    item_count = serializers.SerializerMethodField(read_only=True)
+
+    def get_item_count(self, instance):
+        """get the number of items in this collection"""
+        return WebsiteCollectionItem.objects.filter(website_collection=instance).count()
```

Comment:
This could be an n+1 query but it may not matter. Are there likely to be a lot of `WebsiteCollection`s?

Reply:
I'm not sure, I think at least a few

I wasn't sure actually if we even needed this field, or if it maybe would be better to set it as a value on the model whenever we add or remove a `WebsiteCollectionItem`. I was also wondering if maybe this is something where I would need to do something with the `get_queryset` method on the associated view in order to eliminate possible issues - what do you think?

Reply:
I think I'll just take out the 'num_items' thing for now to keep it simple